client/src/Components/ShareButton.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { StyledShareButton, StyledShareImage, StyledShareText } from '../Styles/style';
+
+class ShareButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { shareButtonClicked: false };
+
+    this.onShareButtonClick = this.onShareButtonClick.bind(this);
+  }
+
+  onShareButtonClick(event) {
+    event.preventDefault();
+    this.setState(prevState => ({ shareButtonClicked: !prevState.shareButtonClicked }));
+  }
+
+  render() {
+    const { shareButtonClicked } = this.state;
+
+    return (
+      <StyledShareButton type="button" onClick={this.onShareButtonClick}>
```

Comment:
Do you need to specify type is button if the styled component is already a button?

Reply:
Removed this one as well!